packages/flutter/lib/src/widgets/navigator_pop_handler.dart

```diff
@@ -81,9 +81,9 @@ class _NavigatorPopHandlerState extends State<NavigatorPopHandler> {
   Widget build(BuildContext context) {
     // When the widget subtree indicates it can handle a pop, disable popping
     // here, so that it can be manually handled in canPop.
-    return PopScope(
+    return PopScope<Object?>(
```

Comment:
I wonder if people will want NavigatorPopHandler itself to have a type parameter, which is passed to PopScope here? I guess it's probably best to save that for another PR if/when it's needed.

Reply:
That was what i did before, but I can't think of a reason you need to access result in `NavigatorPopHandler`

Reply:
If I understand correctly, NavigatorPopHandler is for handling system back gesture, which won't have any result

Reply:
You're right, it shouldn't ever be necessary to set that as far as I can tell.